jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -48,4 +51,7 @@ def get_migrations(self) -> List[Migration]:
                     "update 'type' and 'unicity_id' fields in job from /split-names to /split-names-from-streaming"
                 ),
             ),
+            MigrationAddJobRunnerVerionToCacheResponse(
+                version="20230309141600", description="add 'job_runner_version' field based on 'worker_version' value"
```

Comment:
and delete 'worker_version'

Reply:
I will send 'worker_version' removal in another PR